engine/src/hopeit/app/config.py

```diff
@@ -244,9 +251,13 @@ class EventConfig:
     response_timeout: float = 60.0
     logging: EventLoggingConfig = field(default_factory=EventLoggingConfig)
     stream: EventStreamConfig = field(default_factory=EventStreamConfig)
```

Comment:
It might me needed to split stream settings in two: read_stream and write_stream in order to allow different parameters for each stream for consumer/publisher events

Reply:
Not for now.